core/qt-ble.h

```diff
@@ -23,7 +23,13 @@ class BLEObject : public QObject
 	inline void set_timeout(int value) { timeout = value; }
 	dc_status_t write(const void* data, size_t size, size_t *actual);
 	dc_status_t read(void* data, size_t size, size_t *actual);
-	inline const char *get_name() { return device->btname; }
+	inline dc_status_t get_name(char *res, size_t size)
```

Comment:
From what I can tell, this all looks good. However, note that this is way out of my comfort zone.

My only stylistic comment would be to un-inline this function. I'm not a fan of mixing function declaration and function definition. Moreover, the `inline` keyword is redundant here. These stylistic issues can be addressed in a later commit, though.

Reply:
yeah, let's deal with that later and merge now... mainly to see if that deals with the weird build issue that I still don't understand. I'm actually worried that there is some older include file that is part of the build container and that somehow that gets picked over the one that is built during the GitHub Action run. If that's the case, then I need to fix / update the build container in order to address that :-/

Reply:
This pr is closed, but just commenting on this part anyway for posterity: I agree, but it would probably be best to just make the whole "ioctl()" function be the thing we uninline and make a public function.

But yeah, for this round I just did the minimal changes for just the interface change.